Client/Frontend/Browser/Tab.swift

```diff
@@ -830,6 +832,34 @@ class Tab: NSObject {
     }
 }
 
+extension Tab: UIGestureRecognizerDelegate {
+    // This prevents the recognition of one gesture recognizer from blocking another
+    func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer, shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer) -> Bool {
+        return true
+    }
+    
+    func configureEdgeSwipeGestureRecognizers() {
+        guard let webView = webView else {
+            os_log(.debug, "Tab's edge swipe gesture recognizer was never added. This will affect Tab navigation telemetry!")
+            return
+        }
+        
+        let swipeGesture = UIScreenEdgePanGestureRecognizer(target: self, action: #selector(handleEdgeSwipeTabNavigation(_:)))
+        swipeGesture.edges = .left
+        swipeGesture.delegate = self
+        webView.addGestureRecognizer(swipeGesture)
+    }
+    
+    @objc func handleEdgeSwipeTabNavigation(_ sender: UIScreenEdgePanGestureRecognizer) {
+        if let webView = webView {
```

Comment:
Maybe use `guard let webView = webView else { return }` to reduce arrow-code (`if` in another `if`)? Or combine the two `if` together and put that on multiple lines?